src/shared/managers/SubscriptionManager.ts

```diff
@@ -163,7 +163,8 @@ export class SubscriptionManager {
   }
 
   private async _updatePushSubscriptionModelWithRawSubscription(rawPushSubscription: RawPushSubscription) {
-    const pushModel = await OneSignal.coreDirector.getCurrentPushSubscriptionModel();
+    // This undefined when it shouldn't be
```

Comment:
Can you rephrase the comment?

Reply:
ah ops, ya that problem is solved now, as this is the reason for this PR 😄